src/libs/ReportUtils.ts

```diff
@@ -2992,6 +2992,10 @@ function getReportName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> = nu
         return Localize.translateLocal('parentReportAction.deletedTask');
     }
 
+    if (isGroupChat(report)) {
+        return getGroupChatName(report?.participantAccountIDs ?? [], true, report?.reportID) ?? '';
```

Comment:
- We can safely omit the `report.participantAccountIDs` here.
- Passing `shouldApplyLimit = true` means that we will be truncating a name where we were previously not truncating it for the [ReportSettingsPage.tsx](https://github.com/Expensify/App/pull/40134/files#diff-15fcefa64d91e96ad3f3b080f20e819c6a4fffefabd9712b2cde44bfcdaaa9af). Is it OK?

Reply:
That sounds good to me, thanks. If we figure out that we do need the short name later it's an easy tweak.